examples/autofill-utils/src/autofill-utils.rs

```diff
@@ -107,125 +108,119 @@ enum Command {
     },
 }
 
-fn run_add_address(db: &mut AutofillDb, filename: String) -> Result<()> {
+fn run_add_address(store: Store, filename: String) -> Result<()> {
```

Comment:
These should probably take a reference to the store - we don't need to transfer ownership I don't think. It doesn't matter though - we could always change it later if necessary.